jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    testPathIgnorePatterns: ["/.next/", "/node_modules/"],
+    setupFilesAfterEnv: ["./setupTests.js"],
```

Comment:
Can we move this into `test/lib/setup.js` to keep all tests related code under `test`? The `jest.Timeout` directive in that file currently does not have any impact, so you can remove it. It was added long back when tests were running in dev environment.

Reply:
Okay. I did move all the files under the `test` directory. I have renamed `setup.js` to `jest.config.js` if that's alright.